src/Compiler.ts

```diff
@@ -49,6 +51,8 @@ export function compile(fileNames: string[], options: CompilerOptions): void {
             try {
                 const rootDir = options.rootDir;
 
+                // console.log(`Transpiling ${sourceFile.fileName}...`);
```

Comment:
Either uncomment or remove (I like it to see which file is causing errors).

Reply:
It should probably only print on error then.